### PR TITLE
[IMP] website: introduce `s_quadrant` snippet

### PR DESCRIPTION
**Pull request by xlu-odoo**
This commit adds the new `s_quadrant` snippet.

task-4105045
Part-of: task-4077427

requires: https://github.com/odoo/design-themes/pull/874

Capture d’écran 2024-08-20 à 15 37 59

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website/views/snippets/s_quadrant.xml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_quadrant" name="Quadrant">
+    <section class="s_quadrant">
+        <div class="container-fluid">
+            <div class="row o_grid_mode" data-row-count="14">
+                <div class="o_grid_item o_grid_item_image g-col-lg-6 g-height-6 col-lg-6 d-lg-block d-none o_snippet_mobile_invisible" style="z-index: 1; grid-area: 1 / 1 / 8 / 7; --grid-item-padding-y: 0px; --grid-item-padding-x: 0px;">
```
Comment (Brieuc-brd) [inline addons/website/views/snippets/s_quadrant.xml:8]:
```suggestion
                <div class="o_grid_item o_grid_item_image g-col-lg-6 g-height-7 col-lg-6 d-lg-block d-none o_snippet_mobile_invisible" style="z-index: 1; grid-area: 1 / 1 / 8 / 7; --grid-item-padding-y: 0px; --grid-item-padding-x: 0px;">
```

File: addons/website/views/snippets/s_quadrant.xml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                </div>
+                <div class="o_grid_item o_grid_item_image g-col-lg-6 g-height-7 col-lg-6 d-lg-block d-none o_snippet_mobile_invisible" style="z-index: 4; grid-area: 8 / 7 / 15 / 13; --grid-item-padding-y: 0px; --grid-item-padding-x: 0px;">
+                    <img class="img img-fluid" src="/web/image/website.s_quadrant_default_image_4" alt=""/>
+                </div>
+                <div class="o_grid_item g-height-8 g-col-lg-6 col-lg-6 o_cc o_cc1 rounded justify-content-center" style="z-index: 5; grid-area: 4 / 4 / 12 / 10; --grid-item-padding-y: 32px; --grid-item-padding-x: 32px;">
+                    <h2 style="text-align: center;">Understanding the Innovation</h2>
+                    <p class="lead" style="text-align: center;">Explore how our cutting-edge solutions redefine industry standards. To achieve excellence, we focus on what truly matters to our customers. <br/><br/> Begin by identifying their needs and exceed their expectations.</p>
+                    <p style="text-align: center;">
+                        <a t-att-href="cta_btn_href" class="btn btn-primary btn-lg"><t t-esc="cta_btn_text">Learn more</t></a>
```
Comment (Brieuc-brd) [inline addons/website/views/snippets/s_quadrant.xml:24]:
```suggestion
                        <a t-att-href="cta_btn_href" class="btn btn-primary btn-lg"><t t-out="cta_btn_text">Learn more</t></a>
```

**Review by Brieuc-brd — CHANGES_REQUESTED** [review]
Good job @xlu-odoo, it's clean 👍 
I've left a few comments below ;)

Also :
- As this snippet centers the content, I wonder if it wouldn't be nice to have an image below this block on mobile. Just a thought, what do you think?
Capture d’écran 2024-08-19 à 17 07 53

- The snippet is broken when we switch to columns :
Capture d’écran 2024-08-19 à 17 08 32
I think we should prevent it by hiding the option (check here to see how to do it : https://github.com/odoo/odoo/pull/176232/files#diff-9f41c68ba854940dda3361dc32795e2d7e9fc5b83ef7f8afd35a455b493e9930R649)

**Comment by xlu-odoo** [comment]
> Good job @xlu-odoo, it's clean 👍 I've left a few comments below ;)
> 
> Also :
> 
> * As this snippet centers the content, I wonder if it wouldn't be nice to have an image below this block on mobile. Just a thought, what do you think?
>   Capture d’écran 2024-08-19 à 17 07 53
> * The snippet is broken when we switch to columns :
>   Capture d’écran 2024-08-19 à 17 08 32
>   I think we should prevent it by hiding the option (check here to see how to do it : [#176232 (files)](https://github.com/odoo/odoo/pull/176232/files#diff-9f41c68ba854940dda3361dc32795e2d7e9fc5b83ef7f8afd35a455b493e9930R649))

Excellent idea for the mobile. (using order)
Made all the changes, thanks for the review @Brieuc-brd .

**Review by Brieuc-brd — APPROVED** [review]
Thanks @xlu-odoo !
Just a nitpick below 😬 

Beside that LGTM 👍

**Comment by bso-odoo** [comment]
@robodoo delegate=bvr-odoo

**Comment by bvr-odoo** [comment]
@robodoo r+

**Pull request closed without merging**